tests/api/metrics/test_job.py

```diff
@@ -101,3 +101,12 @@ def test_job_cost(client, workflow_job, headers):
     for line in metrics.text.split("\n"):
         if "github_actions_job_cost_count_total{" in line:
             assert "0.104" in line
+
+
+def test_skipped_job(client, workflow_job, headers):
```

Comment:
To reproduce the issue might need to add some config in the exporter

```suggestion
def test_skipped_job(override_job_config, client, workflow_job, headers):
```